### ISSUE TITLE
IAMPolicy and IAMPolicyMember do not accept projectViewer

### ISSUE THREAD

**Issue by fsommar** [GoogleCloudPlatform/k8s-config-connector#234]
**Describe the bug**

The `IAMPolicy` and `IAMPolicyMember` resources don't accept projectViewer as a member reference.

This is an addendum to a previously closed issue here: https://github.com/GoogleCloudPlatform/k8s-config-connector/issues/208. The 1.13.0 update fixed projectOwner and projectEditor member references, but not projectViewer.

**ConfigConnector Version**
1.13.1

**To Reproduce**

Create a bucket IAM policy using projectViewer to delegate bucket permissions. See original ticket, https://github.com/GoogleCloudPlatform/k8s-config-connector/issues/208, for more information.

*YAML snippets:*

```yaml
apiVersion: iam.cnrm.cloud.google.com/v1beta1
kind: IAMPolicy
metadata:
  name: test-bucket-iampolicy
spec:
  bindings:
  - members:
    - projectViewer:project-name
    role: roles/storage.legacyBucketReader
  resourceRef:
    apiVersion: storage.cnrm.cloud.google.com/v1beta1
    kind: StorageBucket
    name: test-bucket
```

This above YAML should be legal based on the special-casing for member references in bucket permissions. Instead, the CRD regex rejects this since it doesn't explicitly accept the projectViewer special reference kind.

**Comment by maqiuyujoyce** [comment]
Hi @fsommar , thank you for reporting it! We'll fix it soon.

**Comment by jcanseco** [comment]
Hi @fsommar, we just released v1.15.0 which adds support for the `projectViewer` prefix for members in `IAMPolicy` and `IAMPolicyMember`. I'll be closing this issue now, though please feel free to re-open it if you have any further issues!
